src/content/ja/2022/sustainability.md

```diff
@@ -0,0 +1,814 @@
+
+## 人気のフレームワーク、プラットフォーム、CMS
+
+オンラインプラットフォームやCMSツールは、ウェブ上で出版やビジネスを行いたい人々の参入障壁を下げるのに役立ちます。同様に、開発フレームワークやサイトジェネレーターは、ウェブのために構築する人々がプロジェクトを迅速に開始し、一般的な開発問題を解決するデフォルトやソリューションを活用できるようにします。
+
+以下のチャートは、もっとも人気のある5つのeコマースプラットフォーム、CMSツール、サイトジェネレーターツールの中央ページ重量を示しています。
+
+{{ figure_markup(
+  image="median-kilobytes-by-ecommerce.png",
+  caption="eコマース別の中央値キロバイト",
+  description="WooCommerceページはデスクトップで3,048 KB、モバイルで2,827 KBの中央重量があることを示す棒グラフ。Shopifyはデスクトップで2,428 KB、モバイルで2,080 KBです。Squarespace Commerceはデスクトップで3,462 KB、モバイルで3,577 KBです。PrestaShopはデスクトップで2,900 KB、モバイルで2,528 KBです。Magentoはデスクトップで3,317 KB、モバイルで3,093 KBです。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=574238152&format=interactive",
+  sheets_gid="1434449385",
+  sql_file="ecommerce_bytes_per_type.sql"
+) }}
+
+{{ figure_markup(
+  image="median-kilobytes-by-cms.png",
+  caption="CMS別の中央値キロバイト",
+  description="WordPressページはデスクトップで2,559 KB、モバイルで2,314 KBの中央重量があることを示す棒グラフ。Drupalはデスクトップで2,351 KB、モバイルで2,146 KBです。Joomlaはデスクトップで2,799 KB、モバイルで2,495 KBです。Wixはデスクトップで3,172 KB、モバイルで2,158 KBです。Squarespaceはデスクトップで3,462 KB、モバイルで3,577 KBです。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=1499422639&format=interactive",
+  sheets_gid="1561070567",
+  sql_file=""
+) }}
+
+{{ figure_markup(
+  image="static-site-generator-median.png",
+  caption="静的サイトジェネレーターの中央値KB",
+  description="Next.jsページはデスクトップで2,387 KB、モバイルで2,064 KBの中央重量があることを示す棒グラフ。Nuxt.jsはデスクトップで2,877 KB、モバイルで2,210 KBです。Gatsbyはデスクトップで2,049 KB、モバイルで1,731 KBです。Hugoはデスクトップで870 KB、モバイルで1,088 KBです。Jekyllはデスクトップで662 KB、モバイルで781 KBです。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=1545025671&format=interactive",
+  sheets_gid="613315308",
+  sql_file="ssg_bytes_per_type.sql"
+) }}
+
+とくに興味深いのは、リストされたプラットフォーム/ツールの中で、全体の中央値（2,019 KB）よりもモバイルページの重量が少ないのは3つだけであることです。これらはすべて静的サイトジェネレーターカテゴリにあり、とくにHugoとJekyllの場合は、これらのツールが使用されているウェブサイトの種類、主にブログやテキストコンテンツ、JavaScriptへの依存度が少ないことに起因する可能性が高いです。また、SSGはパフォーマンスを念頭に置いて使用されることが多いため、商品理由のみでCMSを使用している平均的なウェブサイトよりもさらに最適化される可能性が高いことも指摘しておきます。
+
+3つのセグメントを横断して見ると、デスクトップとモバイルのページサイズの間に大きな差があることがわかります。詳しく見ると、これはいくつかのプラットフォームがモバイルデバイス用に画像の最適化を行っているためのようです。これを強調するために、デスクトップとモバイルのサイズに大きな違いを示すWixなど、他の人気プラットフォームと比較したCMSカテゴリを見てみましょう。
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th scope="col">CMS</th>
+        <th scope="col">Device</th>
+        <th scope="col">HTML</th>
+        <th scope="col">JavaScript</th>
+        <th scope="col">CSS</th>
+        <th scope="col">Image</th>
+        <th scope="col">Fonts</th>
```

Comment:
Do some of these need translated? For example Device probably should be translated yet? the others are maybe OK as technical terms.

Reply:
That's a challenging question. I've pondered it myself.
However, I believe I have been translating "device" into Japanese, so I will translate it.
https://github.com/HTTPArchive/almanac.httparchive.org/pull/3772/commits/7711605a29947e5dd00924b13c2cf80cd82b5b03